sql/core/src/test/resources/sql-tests/inputs/typeCoercion/native/functionArgumentConversion.sql

```diff
@@ -25,7 +25,7 @@ SELECT array(cast(1 as tinyint), cast(1 as float)) FROM t;
 SELECT array(cast(1 as tinyint), cast(1 as double)) FROM t;
 SELECT array(cast(1 as tinyint), cast(1 as decimal(10, 0))) FROM t;
 SELECT array(cast(1 as tinyint), cast(1 as string)) FROM t;
-SELECT array(cast(1 as tinyint), cast('1' as binary)) FROM t;
+SELECT size(array(cast(1 as tinyint), cast('1' as binary))) FROM t;
```

Comment:
Replace `array(cast(1 as tinyint), cast('1' as binary))` with `size(array(cast(1 as tinyint), cast('1' as binary)))` to avoid binary type with collection.